libs/components/flyout/src/lib/modules/flyout/flyout-media-query.service.ts

```diff
@@ -1,26 +1,23 @@
 import { Injectable } from '@angular/core';
 import { SkyMediaBreakpoints, SkyMediaQueryListener } from '@skyux/core';
 
-import { BehaviorSubject, Subscription } from 'rxjs';
+import { ReplaySubject, Subscription } from 'rxjs';
 
 /**
  * @internal
  */
 @Injectable()
 export class SkyFlyoutMediaQueryService {
-  public get current(): SkyMediaBreakpoints {
+  public get current(): SkyMediaBreakpoints | undefined {
     return this.#_current;
   }
 
-  #currentSubject: BehaviorSubject<SkyMediaBreakpoints>;
+  #currentSubject: ReplaySubject<SkyMediaBreakpoints>;
 
-  #_current = SkyMediaBreakpoints.xs;
+  #_current: SkyMediaBreakpoints | undefined;
 
   constructor() {
-    this.#currentSubject = new BehaviorSubject<SkyMediaBreakpoints>(
-      this.current
-    );
-    this.#currentSubject.next(this.#_current);
+    this.#currentSubject = new ReplaySubject<SkyMediaBreakpoints>(1);
```

Comment:
This initializes the `ReplaySubject` but now it doesn't immediately emit `this.#_current`. Was this intentional?

Reply:
Yes. I changed that based on your comment regarding `xs` and if we knew that it was the right default. `xs` is the initial size of most flyouts but not all. Our flyout code immediately calls `setBreakpointForWidth` in order to set a breakpoing (and then update the subject); however, the previous behavior of initially outputting `xs` was based on the default width and not reality. I'm happy to add the `xs` initial back as to not change things if you'd like.